torch/csrc/jit/codegen/cuda/executor_launch_params.h

```diff
@@ -0,0 +1,116 @@
+#pragma once
+#include <torch/csrc/jit/codegen/cuda/type.h>
+
+namespace torch {
+namespace jit {
+namespace fuser {
+namespace cuda {
+
+class TORCH_CUDA_API LaunchParams {
+ public:
+  LaunchParams(
+      int64_t gdimx = -1,
```

Comment:
Create and pass in a struct instead of lots of parameters?

Reply:
Isn't that just going to change where all these parameters have to be set? Not sure this would be cleaner, do you have a specific idea how to clean it up?